versioned_docs/version-2.x/advanced/theme/add-plp-sorting-method.mdx

```diff
@@ -0,0 +1,107 @@
+---
+sidebar_position: 10
+title: Add PLP sorting method
+description:
+  This guide explains how to add a sorting criteria to your Product List Page
+  (PLP).
+---
+
+import Figure from "@site/src/components/Figure";
+import ContactLink from "@site/src/components/ContactLink";
+import SinceVersion from "@site/src/components/SinceVersion";
+
+<SinceVersion tag="2.27" />
```

Comment:
given no code was changed in 2.x for that, it's wrong. I guess this can just be removed

Reply:
and this PR can target `main`

Reply:
Yep, I think I mixed up the 3 docs I wrote at the same time.

Reply:
Removed the Since tag and updated target branch.